firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -871,6 +878,7 @@ class SettingsTableViewController: ThemedTableViewController {
 
         tableView.register(cellType: ThemedLeftAlignedTableViewCell.self)
         tableView.register(cellType: ThemedSubtitleTableViewCell.self)
+        tableView.register(cellType: ThemedCenteredTableViewCell.self)
```

Comment:
We don't need to register the cell here as it's already registered in `ThemedTableViewController`.